app/templates/mock-login.hbs

```diff
@@ -1,54 +1,37 @@
+      <AuHeading @skin="4">Kies een bestuurseenheid om mee in te loggen.</AuHeading>
+      <p class="au-u-margin-top-small au-u-margin-bottom">
+        {{#let (unique-id) as |id|}}
+        <AuLabel for={{id}}>Bestuurseenheid zoeken</AuLabel>
+        <input id={{id}} class="au-c-input au-c-input--block js-js-input-pattern-bound" placeholder="Aalst, Berchem, &hellip;" value={{this.gemeente}} oninput={{perform this.updateSearch value='target.value'}}>
+        {{/let}}
+      </p>
 
-      <div class="au-o-region">
         <MockLogin as |login|>
           {{#if this.queryStore.isRunning}}
-            <AuLoader @size="small" />
-            <span class="au-u-hidden-visually">Aan het laden</span>
+            <AuLoader @padding="small" />
           {{else}}
-            {{#if login.errorMessage}}
-              <AuAlert @icon="info-circle" @title={{login.errorMessage}}  @skin="error">{{login.errorMessage}}</AuAlert>
+            {{#if this.login.errorMessage}}
+              <AuAlert @alertIcon="alert-triangle" @alertTitle={{this.login.errorMessage}} @alertSkin="warning" />
             {{/if}}
-            <login.each-account @accounts={{this.model}} as |account|>
-                <div class="au-o-box au-o-box--small au-c-card au-u-margin-bottom-small">
-                  <p>
-                  {{account.gebruiker.voornaam}} {{account.gebruiker.achternaam}}
-                  </p>
-                  {{#if (await account.gebruiker.bestuurseenheden.firstObject.alternatieveNaam)}}
-                  <AuHr/>
-                  <p>
-                    <span class="au-c-info-text">Alternatieve naam:</span>
-                    {{#each (await account.gebruiker.bestuurseenheden) as |bestuurseenheid|}}
-                      {{#each bestuurseenheid.alternatieveNaam as |alternatieveNaam|}}
-                        <AuPill>{{alternatieveNaam}}</AuPill>
-                      {{/each}}
-                    {{/each}}
-                  </p>
-                  {{/if}}
-                </div>
-            </login.each-account>
-              <AuDataTableNumberPagination
-                @page={{this.page}}
-                @size={{this.size}}
-                @nbOfItems={{this.model.length}}
-                @total={{this.model.meta.count}}
-                @links={{this.model.meta.pagination}}
-              />
+            {{#each this.model as |account|}}
+              <AuButton class="au-u-flex au-u-flex--row au-u-flex--start au-c-card au-u-margin-bottom-small" @skin="secondary" @width="block"  @size="large" type="button" {{on "click" (fn login.login account.id account.gebruiker.group.id)}}>
+                {{account.gebruiker.voornaam}} {{account.gebruiker.achternaam}}
+              </AuButton>
+            {{/each}}
+            <NumberPagination @page={{this.page}} @size={{this.size}} @nbOfItems={{this.model.length}} @total={{this.model.meta.count}} @links={{this.model.meta.pagination}} />
```

Comment:
Also, I just noticed that the total never updates which might be because we don't include the ember-data-table's serializer mixin yet: https://github.com/mu-semtech/ember-data-table#serializer